### ISSUE TITLE
Initializing JS/LS feautures stays loading

### ISSUE THREAD

**Issue by kusiroll** [microsoft/vscode#99719]
Issue Type: <b>Bug</b>

Initializing JS/LS feautures stays forever

VS Code version: Code 1.45.1 (5763d909d5f12fe19f215cbfdd29a91c0fa9208a, 2020-05-14T08:27:35.169Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.60GB (6.91GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-svgviewer|css|2.0.0

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/)?

**Comment by Marklb** [comment]
I had a problem with JS/TS features never completing. It was the only thing I noticed, because it is the only one that displayed a visible loading message, but the more I looked into it I realized that majority of my extensions were not loading or finishing actions.

`vscode-svgviewer@2.0.0` seems to work fine for me, but I narrowed the problem down to the GistPad extension, `vsls-contrib.gistfs@0.0.68` on both my home and office Windows 10 computers on VSCode `v1.46.0`, but I haven't had time to try debugging what exactly is hanging, or whatever is happening, within that extension.

@mjbvz Unless my problem isn't the same, I tried `v1.47.0-insiders` and got the same result.
